### PR TITLE
Code clean up

### PR DESCRIPTION
**Pull request by rocioferreiro**
## Description

> General resolution of warnings, removed unused code and variables.
## ClubHouse Link

> https://app.clubhouse.io/bibliotecame/story/352/code-cleanup
## Checklist
- [X] I've manually tested the feature and it works as expected
- [X] I've successfully integrated the feature with the API
- [X] This pull request has 'sprint_x' as the base branch
- [X] This pull request has a descriptive title and description
- [X] I have merged the current sprint into my branch before pushing
- [X] I've added the QA Leader as a reviewer

### CODE REVIEW

**Review by FacundoBocalandro — CHANGES_REQUESTED** [review]
Chequear la consola de chrome, donde aparecen los warnings de React. La mayoría son con respecto a las keys. Simplemente es agregar la prop key en los componentes que se renderizan con la función map.

image

File: src/Dashboard/Dashboard.tsx [hunk 1]
```diff
@@ -44,10 +44,6 @@ const Dashboard = () => {
     const [labels, setLabels] = React.useState<String[]>([]);
     const [lineChartData, setLineChartData] = React.useState<number[]>([0,0,0,0,0,0,0,0,0,0,0,0])
 
-    useEffect(() => {
-        getInfo();
-    }, [])
```
Comment (FacundoBocalandro) [inline src/Dashboard/Dashboard.tsx:49]: No saques los [], se manda infinitamente la request.

File: src/Login/VerifyToken.tsx [hunk 2]
```diff
@@ -15,7 +15,7 @@ const VerifyToken = () => {
            .catch(() => {
                history.push("/login?unsuccessfulVerification");
            })
-    }, [])
+    }, )
```
Comment (FacundoBocalandro) [inline src/Login/VerifyToken.tsx]: Dejar los [].

**Review by FacundoBocalandro — CHANGES_REQUESTED** [review]
Ver comentarios

File: src/Dashboard/Dashboard.tsx (hunk 1 above)
Comment (rocioferreiro) [reply src/Dashboard/Dashboard.tsx:49]: arreglado

File: src/Login/VerifyToken.tsx (hunk 2 above)
Comment (rocioferreiro) [reply src/Login/VerifyToken.tsx]: arreglado

**Review by FacundoBocalandro — APPROVED** [review]
LGTM. Great job!

**Review by LucasManza — APPROVED** [review]

**Pull request merged**